.github/workflows/dev.yaml

```diff
@@ -0,0 +1,154 @@
+name: Release to dev
+on:
+  push:
+    branches:
+      - dev-cere
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: [self-hosted, cere-network-large-workers]
+    outputs:
+      version: ${{ steps.version.outputs.version }}
+    steps:
+      - uses: actions/checkout@v3
+      - name: Process version
+        id: version
+        run: |
+          echo "::set-output name=version::dev-${{ github.run_attempt}}-$(git rev-parse --short HEAD)"
+      - name: Configure AWS credentials for push images
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.ORG_AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.ORG_AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-west-2
+      - name: Login to Amazon ECR
+        id: login-ecr
+        uses: aws-actions/amazon-ecr-login@v1
+      - name: Set up Docker Buildx
```

Comment:
Why Build**x**?

Reply:
docker/build-push-action@v2 not works without Buildx